pkg/cmd/cli/cmd/importer/dockercompose.go

```diff
@@ -0,0 +1,182 @@
+package importer
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	"k8s.io/kubernetes/pkg/apimachinery/registered"
+	"k8s.io/kubernetes/pkg/kubectl"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/runtime"
+
+	"github.com/openshift/origin/pkg/client"
+	cmdutil "github.com/openshift/origin/pkg/cmd/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	configcmd "github.com/openshift/origin/pkg/config/cmd"
+	"github.com/openshift/origin/pkg/generate/app"
+	appcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+	"github.com/openshift/origin/pkg/generate/dockercompose"
+)
+
+const (
+	dockerComposeLong = `
+Import a Docker Compose file as OpenShift objects
```

Comment:
it's not creating a new template unless you specify --as-template.  The default behavior is to create objects, which means you're processing a template, it's just that the template syntax is Compose instead of OpenShift.